hardeneks/__init__.py

```diff
@@ -159,24 +219,26 @@ def run_hardeneks(
 
     rules = config["rules"]
 
-    console.rule("[b]Checking cluster wide rules", characters="- ")
-    console.print()
-
     resources = Resources(region, context, cluster, namespaces)
     resources.set_resources()
-    harden(resources, rules, "cluster_wide")
+
+    results = []
+
+    cluster_wide_results = harden(resources, rules, "cluster_wide")
+
+    results = results + cluster_wide_results
 
     for ns in namespaces:
-        console.rule(
-            f"[b]Checking rules against namespace: {ns}", characters=" -"
-        )
-        console.print()
         resources = NamespacedResources(region, context, cluster, ns)
         resources.set_resources()
-        harden(resources, rules, "namespace_based")
-        console.print()
+        namespace_based_results = harden(resources, rules, "namespace_based")
+        results = results + namespace_based_results
+
+    print_consolidated_results(results)
 
     if export_txt:
```

Comment:
Do we want users to be able to add all three export options at once? Or do we only want to allow one at a time?

Reply:
I think it is ok for them to do all three export at once? I did it that way assuming if someone needs to see different outputs but run the tool only once